### ISSUE TITLE
Add requirements.txt or config.yml

### ISSUE THREAD

**Issue by YoniSchirris** [ncoudray/DeepPATH#53]
add a requirements.txt or config.yml of the venv you use to run all the code successfully. an example: TS v2 does not work as tf.app is deprecated, and e.g TS v1.15 does work.

**Comment by Tontorino** [comment]
I agree with the need for a requirements.txt (or other file describing the libs versions) for replication of the experiment. I was able to deduce the versions of some dependencies through the README file [here](https://github.com/ncoudray/DeepPATH/tree/master/DeepPATH_code), reading the version of the dependencies loaded into the scripts sections. However, some dependencies continue without their project versions, making it difficult to replicate the project.

**Comment by ncoudray** [comment]
Sorry - it took a while, but there's one now
